style.css

```diff
@@ -0,0 +1,213 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* intro style */
+.container-fluid {
+  position: relative;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
+  border-bottom-left-radius: 60px;
+  box-shadow: 1px 0 1px #fff;
+}
+
+/* logo style */
+.d-flex1 {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 3rem 0 0;
+}
+
+/* main content style */
+.main {
+  width: 100%;
+  padding: 1rem;
+  background-image: url("./images/header-shapes-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.d-flex2 {
+  display: flex;
+  flex-direction: column;
+}
+
+/* intro style */
+.intro {
+  margin-top: 3.5rem;
+}
+
+h1 {
+  font-size: 3rem;
+  padding-bottom: 1rem;
+}
+
+.intro p {
+  line-height: 2rem;
+}
+
+/* lets connect */
+.connect {
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 1.2rem;
+  text-transform: uppercase;
+  padding: 1rem 0;
+}
+
+.social-icons {
+  display: flex;
+}
+
+.social-icons a {
+  padding: 0 1.5rem 0 0;
+}
+
+/* end of intro style */
+
+/* article styling */
+article {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.section2 {
+  margin: 5rem 1rem 1rem 1rem;
+  background-color: #fff;
+  padding: 1rem;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.section2 .showcase-img {
+  width: 100%;
+}
+
+.section2 h2 {
+  font-size: 2.5rem;
+}
+
+.article {
+  display: flex;
```

Comment:
Good job using `display: flex` a lot. FlexBox is more support in various browsers than CSS Grid :+1: 
Here is a link to "Can I Use", a website we use to check what all browsers support  a feature
[Support for Flexbox in "Can I Use"](https://caniuse.com/flexbox)